### ISSUE TITLE
[Bug Report][3.7.13] v-text-field has two labels rendered for variants filled and solo

### ISSUE THREAD

**Issue by MRR0GERS** [vuetifyjs/vuetify#21025]
### Environment
**Vuetify Version:** 3.7.13
**Vue Version:** 3.5.13
**Browsers:** Firefox 135.0
**OS:** Mac OS 10.15

### Steps to reproduce
- use v-text-field, with single-line false, and either no variant or variant set as filled or solo
- inspect the rendered HTML

### Expected Behavior
there should only be one label tag generated when using no variant or using filled or solo variants

### Actual Behavior
two label tags generated when using no variant or using filled or solo variants

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNrNlE1Lw0AQhv/Ksuc2qR94kFjw4lVRPBkPazLRhcnush9REP+7002pm9pCoaHkEJh35p3JPITMyzd3tspvjcm6APyaFx5ag8LDslSMFd1cGBPDKBpt27WKutLKC6nAbpIxbfVnklg7kdHjbkp+dl5y1tYUXVLgWgquSj5oiC0evvy8kYD1sMQYijdAanuE94DClnzbQAQVfGiswZLt4U9tW5dFnr5ouDWVaGXKnYLlSaM+AoSxTlgplCeb2zFqOqB3EhHqkVCbncOmA3sfPNL/MRau3jNuOsAEItVItNT0f9Z0UJ8V7T/m1w17Bx4GnWbSE9w7hpd6ldtc85Xo73yRJ/efpKusNJ458MHQEr2kEv+ZqYA44xfZIlvw11+hibUd)

### Other comments
This has been an issue since Vuetify 3.0.0 and currently exists up to the latest version; 3.7.13 at the time of the report. This is an accessibility issue. Supporting documentation: https://dequeuniversity.com/rules/axe/4.7/form-field-multiple-labels

**Comment by johnleider** [comment]
We hide the inactive label using aria-hidden. Is this still an issue?

![Image](https://github.com/user-attachments/assets/66fe1a9c-f009-4c61-b8d6-8f442e8f0479)

**Comment by KaelWD** [comment]
We don't switch that to the one that's actually hidden though which might be a problem, when the field is active there is one label hidden with `aria-hidden` and another with `visibility: hidden`.

**Comment by MRR0GERS** [comment]
> We don't switch that to the one that's actually hidden though which might be a problem, when the field is active there is one label hidden with `aria-hidden` and another with `visibility: hidden`.

This seems to be correct; on focus the .v-field__outline > .v-field-label--floating should have the `aria-hidden` toggled to false

**Comment by HalgrimThadeus** [comment]
I noticed this issue has already been assigned, which is great. Addressing this is important for us, too. Thank you for your efforts in resolving it!
